src/WasmDis.ts

```diff
@@ -732,6 +737,11 @@ export class WasmDisassembler {
     if (this._done)
       throw new Error('Invalid state: disassembly process was already finished.')
     while (true) {
+      if (this._maxLines && this._lines.length >= this._maxLines) {
+        this.appendBuffer(';; -- truncated --');
```

Comment:
Can you change this message to be more user friendly, e.g. point to the reason of why truncated?

Reply:
Done